modules/microshift-install-bootc-build-image.adoc

```diff
@@ -71,25 +72,30 @@ RUN systemctl enable microshift-make-rshared.service
 Podman uses the host subscription information and repositories inside the container when building the container image. If the `rhocp` and `fast-datapath` repositories are not available on the host, the build fails.
 ====
 
-. Create a local bootc image by running the following image build command:
+. Configure the following environment variables:
 +
-[source,terminal]
+[source,terminal,subs="attributes+",subs="+quotes"]
 ----
 PULL_SECRET=~/.pull-secret.json
-USER_PASSWD=<your_redhat_user_password> # <1>
-IMAGE_NAME=microshift-4.17-bootc
+USER_PASSWD=_<redhat_user_password>_ # <1>
+IMAGE_NAME=microshift-{product-version}-bootc
```

Comment:
This is not rendered properly.

Reply:
I broke these up so there is one attribute each. Sigh. Still troubleshooting those lines which have two code block attributes needed.